### ISSUE TITLE
Missing alt text

### ISSUE THREAD

**Issue by ciaoaaron** [l1ttle-birdy/yahoo-final#7]
https://github.com/l1ttle-birdy/yahoo-final/blob/2aa4912977420750f43d6b0d8e6bfdea723475f5/index.html#L59

Alt-text is needed for screen reader users. Also be mindful of syntax. There should be a space before `width`.
